packages/query/_src/Query/operations/someOrElse.ts

```diff
@@ -4,7 +4,7 @@
  * @tsplus static effect/query/Query.Aspects someOrElse
  * @tsplus pipeable effect/query/Query someOrElse
  */
-export function someOrElse<B>(def: LazyArg<B>) {
+export function someOrElse<B>(def: B) {
```

Comment:
The `orElse` and `orFail` combinators should continue to be lazy in the values they accept for their constructors. Similar to what we have in `core`.